pve2_api.class.php

```diff
@@ -59,9 +59,17 @@ public function __construct ($hostname, $username, $realm, $password, $port = 80
 		$this->hostname   = $hostname;
 		$this->username   = $username;
 		$this->realm      = $realm;
+		$this->tokenid    = $tokenid;
+		$this->tokensecret= $tokensecret;
```

Comment:
needs an extra space before `=`

Reply:
if you have to bump all these vars to the right a bit for alignment, that's fine 👍